### DISCUSSION TITLE
Styling not loading when canonifyURLS set to "true"

### DISCUSSION THREAD

**Discussion by ChristopherRemde** [thuliteio/doks#1240] [Q&A] [answered]
Hey! First of all thank you a lot for your work on this project, really one of the best documentation themes I've seen! :)

I tried to update my site from version 0.5 to the latest version (1.6) according to your [guide ](https://getdoks.org/upgrade-guides/upgrade-to-v1/). The upgrade seemed to work so far, but I can't seem to get the styling to load correctly on my deployed site, when using the canonifyurl=true setting. Setting it to false lets the styling load, but then my URLs don't work correctly anymore, as it is not hosted on the url root (Maybe related to [#1207](https://github.com/gethyas/doks/discussions/1207))?. Any help with fixing this problem is much appreciated!

The deployed site: https://buildingvolumes.github.io/Unity_Geometry_Sequence_Streaming/
The source code: https://github.com/BuildingVolumes/Unity_Geometry_Sequence_Streaming/tree/main/docs

**Comment by h-enk** [comment, marked as answer]
Hey, thanks!  

Please remove `canonifyURLs = true` — https://discourse.gohugo.io/t/hugo-site-have-no-style-after-deploying-on-github-pages/48900/15


Needs updating: https://docs.gethyas.com/guides/deploy/github/ — I'll get back to you later (bit busy now)

**Comment by ChristopherRemde** [comment]
Hey H-enk, thanks a lot for your quick answer, it's much appreciated!

Disableing canonifyURLs let's the styling load again, but unfortunately most of my links don't work anymore than, as the site doesn't sit at the url root, but rather a subdirectory (https://buildingvolumes.github.io/Unity_Geometry_Sequence_Streaming/ works, but clicking on "Get Started" gets then redirected to the wrong url, without the subdirectory).

Is there any workaround for this? The only thing that would come to my mind is adding the subdirectory to all urls manually, although this would break local testing.

**Reply by h-enk** [reply]
The Get Started URL should be the only one you'd have to override (as you did, I saw). The menu links (main and section) should and are working for me.

To make it work locally the same as in production, you could set in `/config/_default/hugo.toml`: 

```toml
baseurl = "https://localhost/Unity_Geometry_Sequence_Streaming/"
```
See also my example repo: https://github.com/gethyas/doks-gh-pages

**Reply by ChristopherRemde** [reply]
Thank you very much! The menu also works fine for me, but when I link to other sections in the markdown, I also have to add 
/Unity_Geometry_Sequence_Streaming/ before every link. But thanks for the tip with the baseURL, as long as I can test it locally, adding this to every link is fine for me! :)
